src/docs/product/accounts/sso/okta-sso.mdx

```diff
@@ -11,49 +11,93 @@ description: "Set up Okta Single Sign-On (SSO) on Sentry."
 1. From the admin console's sidebar, click **Applications > Applications**, then click on "Add Application".
 
 1. Search for `Sentry` and click "Add", and you will be brought to the setup wizard.
-   
+
    ![Sentry in Okta Gallery of Integrations](okta-gallery.png)
-   
-1. For "1. General Settings", no special settings are needed. 
 
-1. For "2. Sign On Options", fill in these fields as follows: 
-   
+1. For "1. General Settings", no special settings are needed.
+
+1. For "2. Sign On Options", provide the following:
+
    - Base URL: `https://sentry.io`
-      
+
       - Important: There should not be a trailing slash at the end of the URL as it will break the redirect from Okta to Sentry.
```

Comment:
edit to update to current and align with verbiage in other areas of the product:
```suggestion
<Alert level="warning" title="No Trailing Spaces">

Confirm the URL doesn't have a trailing whitespace at the end as it will break the redirect from Okta to Sentry.
 
</Alert>
      
```